paymentsheet/src/main/java/com/stripe/android/paymentelement/EmbeddedPaymentElement.kt

```diff
@@ -560,6 +572,30 @@ class EmbeddedPaymentElement @Inject internal constructor(
         fun onResult(result: Result)
     }
 
+    /**
+     * Describes how you handle row selections in EmbeddedPaymentElement
+     */
+    @ExperimentalEmbeddedPaymentElementApi
+    abstract class RowSelectionBehavior internal constructor() {
+        /**
+         * When a payment option is selected, the customer taps a button to continue or confirm payment.
+         * This is the default recommended integration.
+         */
+        class Default : RowSelectionBehavior()
```

Comment:
After trying to build a similar pattern for IAP, I'm wondering if we should instead turn these into `internal` classes, and expose them via factory methods on `RowSelectionBehavior`

Reply:
Yup! That's right. The benefit would be we have a lot more flexibility on our classes. We can add/remove them, and merchants don't need to know any of the details.

Mostly a future binary compatibility story, it'll be easier to maintain.